sources/packages/backend/libs/integrations/src/services/sfas/sfas-restriction.service.ts

```diff
@@ -108,6 +122,50 @@ export class SFASRestrictionService
     await this.repo.save(restriction, { reload: false, transaction: false });
   }
 
+  /**
+   * Create notifications for newly created legacy student restrictions for the ministry.
+   * @param studentIds student ids for which the legacy restriction notification needs to be sent.
+   * @param entityManager entity manager to execute in transaction.
+   */
+  private async createLegacyRestrictionNotifications(
+    studentIds: number[],
+    entityManager: EntityManager,
+  ): Promise<void> {
+    const auditUser = this.systemUsersService.systemUser;
+    const studentLegacyRestrictions = await entityManager
+      .getRepository(StudentRestriction)
+      .find({
+        select: {
+          student: {
+            birthDate: true,
+            user: { id: true, firstName: true, lastName: true, email: true },
+          },
+        },
+        relations: { student: { user: true } },
+        where: {
+          student: { id: In(studentIds) },
+        },
+      });
+    if (!studentLegacyRestrictions?.length) {
+      // There are no notifications to be sent.
+      return;
+    }
```

Comment:
If the below comment is accepted I do not see a reason to keep this check,
https://github.com/bcgov/SIMS/pull/2910#discussion_r1511943923